modules/graphql/src/endpoints.rs

```diff
@@ -20,12 +20,12 @@ pub fn configure(svc: &mut web::ServiceConfig, db: Database, graph: Arc<Graph>)
         .finish();
 
     svc.service(
-        web::resource("/")
+        web::resource("")
```

Comment:
@jcrossley3, because we're using [web::scope](https://actix.rs/docs/url-dispatch#scoping-routes) it behaves as a prefix before the web::resource. That's actually the issue this PR is all about.